### PR TITLE
fn_artySupport spelling error

### PR DESCRIPTION
**Pull request by ComboTombo**
##What type of PR is this.
1. [x] Bug
2. [x] Enhancement

### What have you changed and why?
Information:
Fixed a spelling error in the artillery support function.
Atry ---> Artillery
Arty ---> Artillery (I know, not a spelling error, but still)

### Please specify which Issue this PR Resolves.
closes #1064 

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the Mission in Singleplayer?
2. [x] Have you loaded the Mission in a Dedicated Server?

### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

********************************************************
Notes:

### CODE REVIEW

File: A3-Antistasi/functions/AI/fn_artySupport.sqf [hunk 1]
```diff
@@ -187,7 +187,7 @@ if (_typeArty == "BARRAGE") then
 	_mrkFinal2 setMarkerShapeLocal "ICON";
 	_mrkFinal2 setMarkerTypeLocal "hd_destroy";
 	_mrkFinal2 setMarkerColorLocal "ColorRed";
-	_mrkFinal2 setMarkerTextLocal "Arty Barrage End";
+	_mrkFinal2 setMarkerTextLocal "Artilley Barrage End";
```
Comment (Lazejun) [inline A3-Antistasi/functions/AI/fn_artySupport.sqf:190]: is the New Typo intended? where did the R go.

Comment (ComboTombo) [reply A3-Antistasi/functions/AI/fn_artySupport.sqf:190]: > is the New Typo intended? where did the R go.

6AM sleep deprived me can say that this is a feature!

Comment (ComboTombo) [reply A3-Antistasi/functions/AI/fn_artySupport.sqf]: Fix the spelling error for the spelling error that came with the fix! Classic Combo...

Comment (Bob-Murphy) [reply A3-Antistasi/functions/AI/fn_artySupport.sqf]: @Lazejun please resolve this issue. Thank you.

Comment (Lazejun) [reply A3-Antistasi/functions/AI/fn_artySupport.sqf]: weird way to ask to approve changes but ok

**Pull request merged**